yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -536,37 +536,55 @@ revision 2019-01-02 {
     considered as completely new on the new branch, and any compatibility information
     should be generated from scratch.
     </t>
-	<t>When a compatibility extension statement is present, that extension
-	statement is the authoritative classification of the backwards compatibility
-	of the change to the statement in the specifed revision. This allows a human
-	author to explicitly communicate the compatibilty and potentially override
-	the rules specified in this document. This is useful in a number of situations
-	including:
+    <t>When a compatibility extension statement is present, that extension
+    statement is the authoritative classification of the backwards compatibility
+    of the change to the statement in the specifed revision. This allows a human
+    author to explicitly communicate the compatibilty and potentially override
+    the rules specified in this document. This is useful in a number of situations
+    including:
     <list style="symbols">
-	  <t>When a tool may not be able to accurately determine the compatibilty of a change. For example, a change in a 'pattern' or 'must' statement can be difficult for a user or tool to determine if it is a compatible change.</t>
-	  <t>When a pattern, range or other statement is changed to more correctly define the server constraint. An example is correcting a pattern that incorrectly included 355.xxx.xxx.xxx as a possible IPv4 address to make it only accept up to 255.xxx.xxx.xxx.</t>
-	  <t>When a vendor module lives in a walled garden where ALL clients are known. In this case a change that is really NBC might not be a problem for the clients: the change is BC for all users.</t>
-	</list>	
+      <t>When a tool may not be able to accurately determine the compatibilty of a change. For example, a change in a 'pattern' or 'must' statement can be difficult for a user or tool to determine if it is a compatible change.</t>
+      <t>When a pattern, range or other statement is changed to more correctly define the server constraint. An example is correcting a pattern that incorrectly included 355.xxx.xxx.xxx as a possible IPv4 address to make it only accept up to 255.xxx.xxx.xxx.</t>
+    </list> 
     <t>Nothing about the compatibilty of a YANG statement can be inferred
-    from the absence of a compatibility extension statement. A leaf, for example, 
-    could be changed from a string type to a uint8 type.
-    Even if there is no rev:nbc-change-at substatement against that leaf,
-    the leaf is still considered as having a non-backwards-compatible change.
+    from the absence of a compatibility extension statement.
```

Comment:
updated proposal:    <t>Nothing about the backwards compatibility of a YANG statement is implied by the absence of a compatibility extension statement.  Hence, the YANG statement must be compared between the two revisions to determine the backwards compatibility.